composer.json

```diff
@@ -17,25 +17,27 @@
     "require": {
         "php": ">=7.4",
         "endroid/qr-code-bundle": "3.4",
-        "common-gateway/template-bundle": "^0.0.2",
-        "commongateway/corebundle": "dev-master"
+        "common-gateway/template-bundle": ">=0.0",
+          "commongateway/corebundle": "^1.1.96",
+        "common-gateway/zgw-bundle": ">=0.1"
```

Comment:
_If possible, requiring the latest possible zgw-bundle version (or higher) might be better than just 0.1 or higher_